### PR TITLE
Centralize web app name

### PR DESCRIPTION
**Pull request by wsedor**
# Description
- centralized the web app name
- changed top level pages to reflect correct name
- changed start script name to be more generic to shield against future name changes

# Testing
Screenshot 2023-11-23 103740

# Reviewers
@jojoboateng @markowusu @KristenAP

### CODE REVIEW

File: streamlitDemo/pages/4_💡_BrillAI_1st_Demo.py
Comment (wsedor) [inline streamlitDemo/pages/4_💡_BrillAI_1st_Demo.py:1]: this was just a rename of the file and adding in the web app name however github deemed it as a file deletion and addition as opposed to a rename

File: streamlitDemo/pages/5_💡_BrillAI_2nd_Demo.py
Comment (wsedor) [inline streamlitDemo/pages/5_💡_BrillAI_2nd_Demo.py:1]: this was just a rename of the file and adding in the web app name however github deemed it as a file deletion and addition as opposed to a rename

File: streamlitDemo/pages/7_✨_Brilla_AI_Live_Quiz.py
Comment (wsedor) [inline streamlitDemo/pages/7_✨_Brilla_AI_Live_Quiz.py:1]: here github correctly detects the rename and displays it as such

File: streamlitDemo/start_webapp.py
Comment (wsedor) [inline streamlitDemo/start_webapp.py:1]: renamed this file as well so it's not tied to the name of the webapp. this shield from potential future name changes

File: streamlitDemo/start_webapp.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 headerString = '''
 ##################################################
-#########         BrillAI Web App        #########
+#########         Brilla AI Web App        #########
```
Comment (wsedor) [inline streamlitDemo/start_webapp.py:6]: code here doesn't import `APP_NAME` from `utility.py` as that will result in an error. `start_webapp.py` is supposed to build all the dependencies ad start the app so importing from `utility.py` won't work as libraries used in that file haven't yet been installed

**Review by markowusu — APPROVED** [review]
Good approach to make the app name centralised. We can create a task to remove that from the utilities file and have a separate folder and  json file for anything that has to do with text. Reducing the hard coding of text in the application.  This is a step in the right direction but we must apply it to every aspect of the application too as well.

**Comment by wsedor** [comment]
> Good approach to make the app name centralised. We can create a task to remove that from the utilities file and have a separate folder and json file for anything that has to do with text. Reducing the hard coding of text in the application. This is a step in the right direction but we must apply it to every aspect of the application too as well.

@markowusu yes. I had a task at some point to do that but skipped my mind for a while. Let's plan to tackle that as well once we resume

**Review by jojoboateng — APPROVED** [review]

**Pull request merged**